addons/website_livechat/controllers/main.py

```diff
@@ -36,3 +34,13 @@ def channel_rating(self, channel, **kw):
             'percentage': percentage
         }
         return request.render("website_livechat.channel_page", values)
+
+    @http.route('/livechat/history_pages', type="json", auth="public")
+    def history_pages(self, pid, channel_id, page_history=None):
+        page_history = map(lambda url: "<a target='_blank' href='%s'>%s</a>" % (url, url), page_history)
```

Comment:
Make a method on `mail.channel` to broadcast. It's strange to see the bus in a controller.